pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -1076,7 +1076,7 @@ class HdRprApiImpl {
         RecursiveLockGuard rprLock(g_rprAccessMutex);
 
         // return Converged if max samples is reached
-        if (m_iter > m_maxSamples) {
+        if (m_iter >= m_maxSamples) {
```

Comment:
If we set max samples to 256 and we have already done 256 iterations we should stop render